Makefile

```diff
@@ -35,6 +35,11 @@ NOSE_ARGS ?= --stop
 
 ZIP_TYPE := $(shell if [ "$(ZIP_DEV)" == "true" ]; then echo '-dev'; else echo ''; fi)
 
+CARGO_HOME := /root/.cargo
+SCCACHE_CACHE_SIZE" := 40G,
+SCCACHE_DIR := /root/.cache,
+RUSTC_WRAPPER := /root/.cargo/bin/sccache
```

Comment:
This is going to break people who run the makefile without sccache

Reply:
Initially, I put these in the provisioner script. Perhaps I misunderstood when you said to move it to the profile. Where should these live?

Reply:
I've moved this into the bash_profile on the VM. I checked the process as it was running and verified that the environment variables are set for the process:

```
[root@adm 30210]# cat environ
XDG_SESSION_ID=24HOSTNAME=adm.localSHELL=/bin/bashTERM=vt100HISTSIZE=1000SSH_CLIENT=10.0.2.2 52981 22OLDPWD=/home/vagrantUSER=rootSUDO_USER=vagrantSUDO_UID=1000USERNAME=rootMAIL=/var/spool/mail/rootPATH=/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin:/root/binCARGO_HOME=/root/.cargoPWD=/integrated-manager-for-lustreRUSTC_WRAPPER=/root/.cargo/bin/sccacheLANG=en_US.UTF-8SCCACHE_CACHE_SIZE=40GHISTCONTROL=ignoredupsSHLVL=3HOME=/rootSUDO_COMMAND=/bin/bash -lLOGNAME=rootSSH_CONNECTION=10.0.2.2 52981 10.0.2.15 22LESSOPEN=||/usr/bin/lesspipe.sh %sSCCACHE_DIR=/root/.cacheXDG_RUNTIME_DIR=/run/user/1000SUDO_GID=1000_=/bin/make
```